sdk/communication/azure-communication-chat/src/test/java/com/azure/communication/chat/ChatThreadAsyncClientTest.java

```diff
@@ -299,8 +299,6 @@ public void canSendThenListReadReceiptsWithResponse() throws InterruptedExceptio
 
         chatThreadClient.sendReadReceiptWithResponse(response.getId()).block();
 
-        Thread.sleep(500);
-
         PagedIterable<ReadReceipt> readReceiptsResponse = new PagedIterable<ReadReceipt>(chatThreadClient.listReadReceipts());
 
         // process the iterableByPage
```

Comment:
Do we also need to include lines 302-309 in the if playbackmode clause? I think the response was returning with 404

Reply:
I believe the 404 was happening sporadically when the message was sent, and we then send a readreceipt
-  chatThreadClient.sendReadReceiptWithResponse(response.getId())
At least that was the line that failed 1 time in the .NET sdk. We could leave this one as is for now, but if we see that happening I agree with @minnieliu .
In the .NET version, I took the entire sendmessage/sendreadreceipt, getreadreceipts out of the live workflow into the Playback mode to avoid flakiness